src/pages/index.js

```diff
@@ -65,22 +136,54 @@ addNewCardButton.addEventListener("click", () => {
   newCardPopup.openPopup();
 });
 
+avatarPhoto.addEventListener("click", () => {
+  changeAvatarPopupValidator.resetValidation();
+  changeAvatarPopup.openPopup();
+});
+
 const editFormValidator = new FormValidator(settings, profileEditForm);
 editFormValidator.enableValidation();
 
 const addFormValidator = new FormValidator(settings, addCardForm);
 addFormValidator.enableValidation();
 
+const changeAvatarPopupValidator = new FormValidator(settings, avatarPhotoForm);
+changeAvatarPopupValidator.enableValidation();
+
 const newCardPopup = new PopupWithForm("#card-add-modal", submitCardAdd);
 newCardPopup.setEventListeners();
 
 const editProfilePopup = new PopupWithForm("#edit-modal", submitProfileForm);
 editProfilePopup.setEventListeners();
 
+const changeAvatarPopup = new PopupWithForm(
+  "#change-avatar-modal",
+  submitAvatarLink
+);
+changeAvatarPopup.setEventListeners();
+
 const imagePopup = new PopupWithImage("#picture-modal");
 imagePopup.setEventListeners();
 
-const userInfoClass = new UserInfo({
-  userName: ".profile__title",
-  userJob: ".profile__description",
+const cardDeletePopup = new PopupWithConfirmation("#delete-card-modal");
+cardDeletePopup.setEventListeners();
+
+api
+  .getInitialCards()
+  .then((result) => {
+    cardSection.renderItems(result);
+  })
+  .catch((err) => {
+    console.error(err);
+  });
+
+api.getCurrentUserInfo().then((result) => {
+  userInfoMain.setUserInfo({
+    name: result.name,
+    job: result.about,
+  });
+  userInfoMain.setAvatarUrl(result.avatar);
 });
```

Comment:
You need to `catch` possible errors at the very **end** of any server request.

**Please, correct it everywhere in the code**